### PR TITLE
curl: add --fail in example

### PR DESCRIPTION
**Pull request by janisz**
- [x] The page (if new), does not already exist in the repository.
- [x] The page is in the correct platform directory (`common/`, `linux/`, etc.)
- [x] The page has 8 or fewer examples.
- [x] The PR title conforms to the recommended [templates](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#commit-message).
- [x] The page follows the [content guidelines](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#guidelines).
- [x] The page description includes a link to documentation or a homepage (if applicable).

### CODE REVIEW

**Comment by navarroaxel** [comment]
Hi @janisz, welcome and thanks for your submission. The pages can have up to 8 examples. Do you thinks we should replace or edit a current example with this new arguments?

**Comment by janisz** [comment]
@navarroaxel Fixed, I edited current line

**Comment by janisz** [comment]
https://daniel.haxx.se/blog/2021/02/11/curl-fail-with-body/

**Review by sbrl — COMMENTED** [review]
This makes this example rather long.  Furthermore, I can't find `--fail-with-body` in the `curl` man page?

**Comment by pixelcmtd** [comment]
> I can't find `--fail-with-body` in the `curl` man page?

@sbrl Then you're probably running an out-of-date distro.

```
       --fail-with-body
              (HTTP) Return an error on server errors where the HTTP response code is 400 or greater). In normal cases when
              an HTTP server fails to deliver a document, it returns an HTML document stating so (which often also describes
              why and more). This flag will still allow curl to output and save that content but also to return error 22.

              This is an alternative option to -f, --fail which makes curl fail for the same circumstances but without saving
              the content.

              See also -f, --fail. Added in 7.76.0.
```

**Comment by marchersimon** [comment]
I think we could add it to the second example, which is still rather short.

**Comment by janisz** [comment]
Maybe a reason why I think it's important.

Recently I worked withers script that was using curl. Unfortunately when you don't pass -f option curl returns 0 even if server returned error. That takes me some time to debug since it I must check all steps of my script to detect where it fails.

**Comment by marchersimon** [comment]
Still, we should add it to an example, which isn't too long yet. Or we could remove another one if it's not that important.

**Comment by sbrl** [comment]
I'm using Ubuntu 21.04, @janisz

**Comment by marchersimon** [comment]
@janisz why did you close this? I think documenting the flag makes sense, we just have to find a way to fit in into the already full page.

**Comment by janisz** [comment]
Until recent curl versions go into mainstream repositories adding --fail-with-body will only confuse people

File: pages/common/curl.md [hunk 1]
```diff
@@ -12,9 +12,9 @@
 
 `curl --remote-name {{http://example.com/filename}}`
 
-- Download a file, following location redirects, and automatically continuing (resuming) a previous file transfer:
+- Download a file, following location redirects, and automatically continuing (resuming) a previous file transfer and return error on server error:
```
Comment (marchersimon) [inline pages/common/curl.md:15]:
```suggestion
- Download a file, following location redirects, and automatically continuing (resuming) a previous file transfer and return an error on server error:
```

**Review by marchersimon — APPROVED** [review]
Thanks @janisz

**Review by sbrl — APPROVED** [review]
This complicates an already full page, but in this instance I think it might be ok.

**Pull request merged**